addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -6,14 +6,73 @@ class AccountTax(models.Model):
 
     _inherit = 'account.tax'
 
+    l10n_ar_type_tax_use = fields.Selection(
+        selection=[
+            ('sale', 'Sales'),
+            ('purchase', 'Purchases'),
+            ('none', 'Other'),
+            ('supplier', 'Vendor Payment Withholding'),
+            ('customer', 'Customer Payment Withholding')
+        ],
+        compute='_compute_l10n_ar_type_tax_use', inverse='_inverse_l10n_ar_type_tax_use',
+        string="Argentina Tax Type"
+    )
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
-        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
-
+        selection=[('supplier', 'Vendor Payment'), ('customer', 'Customer Payment')],
+        string="Argentina Withholding Payment Type",
+        help="Withholding tax for supplier or customer payments.")
+    l10n_ar_tax_type = fields.Selection(
+        string='WTH Tax',
+        selection=[
+            ('earnings', 'Earnings'),
+            ('earnings_scale', 'Earnings Scale'),
+            ('iibb_untaxed', 'IIBB Untaxed'),
+            ('iibb_total', 'IIBB Total Amount'),
+        ]
+    )
     l10n_ar_withholding_sequence_id = fields.Many2one(
-        'ir.sequence', 'Withholding Number Sequence', copy=False, check_company=True,
+        'ir.sequence',
+        string='Sequence',
```

Comment:
```suggestion
        string='WTH Sequence',
```
@jjscarafia what do you think to change it like this?